docs/sources/k6-studio/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+weight: 950
+title: k6 Studio
+hero:
+  title: Grafana k6 Studio
+  level: 1
+  image: /media/docs/k6/GrafanaLogo_k6Studio_orange.svg
+  width: 100
+  height: 100
+  description: Grafana k6 Studio is an open-source desktop application for Mac and Windows designed to help you generate k6 test scripts.
```

Comment:
should we add `(linux coming soon)` ?

Reply:
We try to avoid adding any information to the docs that's not accurate to the current version of the product. It's a general best practice for documentation, there are more details on this page from the Google Developer Style Guide: https://developers.google.com/style/timeless-documentation 🤓